06-release-engineering-analytics.Rmd

```diff
@@ -309,9 +315,396 @@ iteratively and planned for four weekly milestones.
 
 <!-- TODO -->
 
-## Appendix
+## Raw extracted data
```

Comment:
https://github.com/nsalminen/software-analytics-book/pull/8#discussion_r221455592

Change "Raw extracted data" to "Extracted Data"?

Reply:
The raw part does not really matter, as long as it mentions "extracted" instead of "synthesized".

Reply:
Okay, then I'll leave it like this :slightly_smiling_face: Thanks again for the feedback! :smile: